AEPServices/Tests/utility/PersistentHitQueueTests.swift

```diff
@@ -234,9 +234,8 @@ class ControllableHitProcessor: HitProcessing {
 
     func processHit(entity: DataEntity, completion: @escaping (Bool) -> Void) {
         processedHits.append(entity)
-        queue.sync {
-            completion(_hitResult)
-        }
+        var hitResult = _hitResult
```

Comment:
Can you remove this line so that `completion(hitResult)` directly reads the boolean using the property defined in the class?